src/TelemetryChannels/ServerTelemetryChannel/Shared/SamplingTelemetryProcessor.cs

```diff
@@ -22,6 +23,8 @@ public sealed class SamplingTelemetryProcessor : ITelemetryProcessor
         private const string RequestTelemetryName = "Request";
         private const string TraceTelemetryName = "Trace";
 
+        private const string SamplingRateMetricName = "Sampling Rate (Preview)";
```

Comment:
Why do we need this metric? It is not in design spec.

Also here is the patter we recommend - chain multiple sampling processors to set various sampling algorithms for different types: https://github.com/SergeyKanzhelev/TR24DEVCT304/blob/master/ApplicationInsightsDataROI/demo2.cs#L43-L65

When you have a chain like this this approach will send contradicting metrics.